mne_bids/mne_bids.py

```diff
@@ -716,6 +716,7 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
 
     # set the raw file name to now be the absolute path to ensure the files
     # are placed in the right location
+    bids_fname = bids_fname.replace('/', op.sep)
```

Comment:
why not just do it up there using op.join?

Reply:
see my other comment/this line is to revert the fixing of `os.sep` to `/` back to `os.sep` (I see I have a typo which technically doesn't matter but it would be better to be consistent)
In testing it seems that this doesn't matter, but I think it be better to return the file to be os-agnostic just in case

Reply:
yes I got it now. But I think it's better to do the other way round. That is bids_fname.replace(op.sep, '/') when passing to _scans_tsv